plugins/CorePluginsAdmin/angularjs/plugins/plugin-management.directive.js

```diff
@@ -25,6 +25,23 @@
 
                     var uninstallConfirmMessage = '';
 
+                    $('.uploadPlugin').click(function (event) {
```

Comment:
We could maybe move that stuff into a new uploadPluginForm directive. Having the javascript directly in the template doesn't sound good to me.